tests/middleware/test_trace_logging.py

```diff
@@ -40,8 +42,8 @@
         },
     },
     "loggers": {
-        "": {"handlers": ["default"], "level": "INFO"},
-        "uvicorn.error": {"handlers": ["default"], "level": "INFO", "propagate": False},
+        "uvicorn": {"handlers": ["default"], "level": "INFO"},
+        "uvicorn.error": {"level": "INFO"},
```

Comment:
There were two problems here:

* The test config overrides the `root` logger.
* It sets `propagate: False` on `uvicorn.error`, which is why when the new test runs after the others, `uvicorn.error` won't be captured. It _could_ be captured without changing the test config — I'd have to do the same trick than for `uvicorn.access` — but that sounds a bit absurd.

I think given that the test trace config aims at focusing on `uvicorn.asgi` in particular, keeping the rest in line with `LOGGING_CONFIG` sounds about right.